tests/store/artifact/test_azure_blob_artifact_repo.py

```diff
@@ -96,8 +96,10 @@ def test_list_artifacts_single_file(mock_client):
     assert repo.list_artifacts("file") == []
 
 
-def test_list_artifacts(mock_client):
-    repo = AzureBlobArtifactRepository(TEST_URI, mock_client)
+@pytest.mark.parametrize("root_path", ["some/path", "some/path/"])
+def test_list_artifacts(mock_client, root_path):
+    repo = AzureBlobArtifactRepository(os.path.join(TEST_BLOB_CONTAINER_ROOT, root_path),
```

Comment:
Can you replace `os.path.join` with `posixpath.join` to prevent this test from failing in windows?

Reply:
and can you apply `black` by running this command?

```
black --line-length=100 --exclude=mlflow/protos .
```

Reply:
Updated!